code/game/objects/effects/aliens.dm

```diff
@@ -158,13 +153,16 @@
 	gender = PLURAL
 	name = "resin floor"
 	desc = "A thick resin surface covers the floor."
-	icon_state = "weeds"
 	anchored = 1
 	density = 0
-	layer = 2
+	layer = TURF_LAYER + 0.09

```

Comment:
stealth bugfix to prevent cult turf overlays from going over the weeds because it looks real odd for like half a second